Sources/DataPipeline/DataPipelineImplementation.swift

```diff
@@ -36,6 +38,26 @@ class DataPipelineImplementation: DataPipelineInstance {
             // if the device token exists, pass it to the plugin and ensure device attributes are updated
             addDeviceAttributes(token: existingDeviceToken)
         }
+
+        // plugin to publish data pipeline events
+        analytics.add(plugin: DataPipelinePublishedEvents(diGraph: diGraph))
+
+        // subscribe to journey events
+        subscribeToJourneyEvents()
```

Comment:
I don't find comments like this valuable in the code. From reading the lines of code, I can see that a plugin is being added and journeys events are being subscribed to. 

If a comment is added, I would find them valuable if the comments mention why we are adding a plugin and why we are subscribing to journeys events.

Reply:
+1, please clean up the comments before merging